dev-requirements.txt

```diff
@@ -3,3 +3,4 @@ pep8==1.6.1
 pyflakes==0.8.1
 mccabe==0.3
 flake8==2.3.0
+selenium==2.44.0
```

Comment:
Move this to `requirements.txt` because that's where we have all other dependencies for running tests. 
Like `mock` and `nose` etc. 

This file (`dev-requirements.txt`) are for extra things that you don't have to necessarily run to make a patch. However, it's just highly highly recommended that you check your code with these tools. But not a must. 

What do you think, should we split up `requirements.txt` into two files? One for running the server and one for running tests? E.g. `requirements.txt` (to be used on servers) and `test-requirements.txt` to be used by travis and local developers.

I like the separation because it means less things needs to be installed on the production servers but it's also nice that there's just one big file you need to look at.